nss-day-cohort-48/fun-website-1-digital-drops
# added the "About Birds" title

##To Test

1. Pull this code
2. git clone the SSH
3. cd into the clone
4. git checkout thejmdw-birds
5. Verify that there is a About Birds title and it's in the right place.
